src/app/ipv/middleware.js

```diff
@@ -255,7 +255,7 @@ module.exports = {
   // https://govukverify.atlassian.net/browse/PYIC-4859.
   updateJourneyState: async (req, res, next) => {
     try {
-      const allowedActions = ["/journey/end"];
+      const allowedActions = ["/journey/end", "/journey/back"];
```

Comment:
Will probably need to refactor this when the validation work gets merged in, but I think it's an ok solution for now

Reply:
Yep I have tested the /journey/end for that scenario, it makes this logic safer because we can then only allow that event on the specific page that this route is built for.

I will have the routing to pass through the current page, but this is higher priority for the time being.

Hopefully once the validation goes in, then we won't be able to just submit a back event for any page 😄

Reply:
Ok - I think the comment above this method is no longer true with this change, so we probably need to update that, and update the validation PR accordingly (so it doesn't pass a hardcoded `pyi-f2f-delete-details`).